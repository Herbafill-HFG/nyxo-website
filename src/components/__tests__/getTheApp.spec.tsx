import React from "react"
import renderer from "react-test-renderer"

import GetTheApp from "../GetAppBanner/index"

describe("Get the app", () => {
  it("renders correctly", () => {
    const getApp = renderer.create(<GetTheApp />).toJSON()
    expect(getApp).toMatchSnapshot()
  })
})
