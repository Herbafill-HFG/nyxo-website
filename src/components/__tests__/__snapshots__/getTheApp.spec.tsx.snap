// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Get the app renders correctly 1`] = `
<div
  className="sc-AxjAm jfxQlI"
>
  <h2
    className="sc-AxhUy izZJGo"
  >
    Get the app
  </h2>
  <p
    className="sc-AxhCb cxKEKE"
  >
    Download the app for free and sleep better starting tonight.
  </p>
  <div
    className="sc-AxirZ jzSnfk"
  >
    <a
      className="sc-AxiKw dlpYyz"
      href="https://apps.apple.com/app/nyxo/id1440417031"
      onClick={[Function]}
      rel="noopener noreferrer"
      target="_new"
    >
      <img
        alt="App Store"
      />
    </a>
    <a
      className="sc-AxiKw dlpYyz"
      href="https://play.google.com/store/apps/details?id=fi.nyxo.app"
      onClick={[Function]}
      rel="noopener noreferrer"
      target="_new"
    >
      <img
        alt="Google Play"
      />
    </a>
  </div>
</div>
`;
